client/src/components/Users/Auth/PasswordInput.tsx

```diff
@@ -59,20 +75,31 @@ function PasswordInput({
     setShowPassword(!showPassword);
   };
 
+  let additionalProps = {};
+
+  if (onBlur) {
+    additionalProps = {
+      onBlur,
+    };
+  }
+
```

Comment:
Just a recommendation, but non-blocking, I think we could wrap this in a function to help with readability

Reply:
Makes sense.  I've updated this so this is wrapped in a function.